### DISCUSSION TITLE
Libre 2 Direct not connecting via Bluetooth

### DISCUSSION THREAD

**Discussion by jrussell88** [NightscoutFoundation/xDrip#3592] [Q&A]
Yesterday I started a new Libre 2 Plus sensor, using the Direct option, but XDrip+ didn't get beyond waiting for the first readings for calibration. 

System Status showed the new sensor under 'Bluetooth Device', and I NFC scanned it repeatedly without success. Oop shows no data.

It connects to LibreLink on the other phone I used to start it - BT is off on that device and when I turned it on, it received data. I can scan it with LibreLink. 

I've rebooted the XDrip+ phone, stopped and started the sensor, restarted the collector, and finally tried 'Forget Device'. 

System Status shows 'Bluetooth Device: None Set' and the XDrip+ home page has a red message:
"First use the menu to scan for your BT device or change the Data Source in Settings!"

I ran the Bluetooth Scan, found the sensor again and connected to it. Now the 'Collecting Initial Readings' screen is up, and says 'No data received yet' despite multiple NFC scans.

Can anybody suggest how to get it connected?

**Comment by PhilTU** [comment]
For Libre 2+ UK you need OOP2 from end of May this year and similar dated nightly build of xDrip+. Sensor code is C6.
